### PR TITLE
Add pulsar 2.8.1 release note

### PR DESCRIPTION
**Pull request by hangc0276**
### Motivation
Pulsar 2.8.1 has been release, add the release note.

### Modification
1. add Pulsar 2.8.1 release note.

### CODE REVIEW

File: site2/website/release-notes.md [hunk 1]
```diff
@@ -1,6 +1,247 @@
...
+- Remove unused listener to reduce creating executor pool [#11215](https://github.com/apache/pulsar/pull/11215)
+
+### Pulsar IO
+- Make KafkaSourceRecord ack() non-blocking to avoid deadlock [#11435](https://github.com/apache/pulsar/pull/11435)
+- Allow Sinks to use native AVRO and JSON [#11322](https://github.com/apache/pulsar/pull/11322)
+- Refine the key in redis sink when key is null [#11192](https://github.com/apache/pulsar/pull/11192)
+- Change the nar package name for pulsar-io-kafka-connect-adaptor [#10976](https://github.com/apache/pulsar/pull/10976)
+
+
+
```
Comment (codelipenghui) [inline site2/website/release-notes.md:243]: Remove the lines.

**Review by codelipenghui — APPROVED** [review]
Great work.

**Review by k2la — APPROVED** [review]

File: site2/website/release-notes.md [hunk 2]
```diff
@@ -1,6 +1,245 @@
...
+- Fix publish_time not set error when broker entry metadata enable without AppendBrokerTimestampMetadataInterceptor [#11014](https://github.com/apache/pulsar/pull/11014)
+- Fix parseMessageMetadata error cause by not skip broker entry metadata [#10968](https://github.com/apache/pulsar/pull/10968)
+- Fix issue where Key_Shared consumers could get stuck [#10920](https://github.com/apache/pulsar/pull/10920)
+- Fix throwable exception not thrown [#10863](https://github.com/apache/pulsar/pull/10863)
+- Handle multiple topic creation for the same topic-name in broker [#10847](https://github.com/apache/pulsar/pull/10847)
+- Add REST API to enable or disable replicated subscriptions [#10790](https://github.com/apache/pulsar/pull/10790)
+- Fix issue that message ordering could be broken when redelivering messages on Key_Shared subscription [#10762](https://github.com/apache/pulsar/pull/10762)
+- Fix set-publish-rate when using preciseTopicPublishRateLimiterEnable=true [#10384](https://github.com/apache/pulsar/pull/10384)
+- Upgrade BookKeeper version to resolve the BouncyCatle issue [#11759](https://github.com/apache/pulsar/pull/11759)
```
Comment (Jennifer88huang) [inline site2/website/release-notes.md:21]: BouncyCatle or BouncyCastle?

File: site2/website/release-notes.md [hunk 3]
```diff
@@ -1,6 +1,245 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.8.1 &mdash; 2021-09-10 <a id=“2.8.1”></a>
+
+### Broker
+- Fix all web threads get stuck when deleting the namespace [#11596](https://github.com/apache/pulsar/pull/11596)
```
Comment (Jennifer88huang) [inline site2/website/release-notes.md:7]: Fix the issue that all web threads get stuck when deleting ....

File: site2/website/release-notes.md [hunk 4]
```diff
@@ -1,6 +1,245 @@
...
+- Handle multiple topic creation for the same topic-name in broker [#10847](https://github.com/apache/pulsar/pull/10847)
+- Add REST API to enable or disable replicated subscriptions [#10790](https://github.com/apache/pulsar/pull/10790)
+- Fix issue that message ordering could be broken when redelivering messages on Key_Shared subscription [#10762](https://github.com/apache/pulsar/pull/10762)
+- Fix set-publish-rate when using preciseTopicPublishRateLimiterEnable=true [#10384](https://github.com/apache/pulsar/pull/10384)
+- Upgrade BookKeeper version to resolve the BouncyCatle issue [#11759](https://github.com/apache/pulsar/pull/11759)
+- Fix getPreviousPosition NPE [#11621](https://github.com/apache/pulsar/pull/11621)
+- Remove duplicated configuration [#11283](https://github.com/apache/pulsar/pull/11283)
+- Source tarball: apply executable file permissions to shell scripts [#11858](https://github.com/apache/pulsar/pull/11858)
+- Fix java_test_functions build failed [#11829](https://github.com/apache/pulsar/pull/11829)
+- Fix generate javadoc for kafka-connect-adaptor failed [#11807](https://github.com/apache/pulsar/pull/11807)
```
Comment (Jennifer88huang) [inline site2/website/release-notes.md:26]: I was wondering do we just copy the PR titles or improve the readability of the sentences as well?

**Review by Anonymitaet — APPROVED** [review]

File: site2/website/release-notes.md (hunk 3 above)
Comment (Anonymitaet) [inline site2/website/release-notes.md:7]:
```suggestion
- Fix the issue of all web threads get stuck when deleting a namespace [#11596](https://github.com/apache/pulsar/pull/11596)
```

File: site2/website/release-notes.md [hunk 5]
```diff
@@ -1,6 +1,245 @@
...
 
+### 2.8.1 &mdash; 2021-09-10 <a id=“2.8.1”></a>
+
+### Broker
+- Fix all web threads get stuck when deleting the namespace [#11596](https://github.com/apache/pulsar/pull/11596)
+- Improve error logs in BacklogQuotaManager [#11469](https://github.com/apache/pulsar/pull/11469)
+- Solve the issue of precise rate limiting does not take effect [#11446](https://github.com/apache/pulsar/pull/11446)
+- Fix replicated subscriptions direct memory leak [#11396](https://github.com/apache/pulsar/pull/11396)
+- Change ContextClassLoader to NarClassLoader in ProtocolHandler [#11276](https://github.com/apache/pulsar/pull/11276)
+- Fix ledger rollover scheduled task [#11116](https://github.com/apache/pulsar/pull/11116)
```
Comment (Anonymitaet) [inline site2/website/release-notes.md:12]:
```suggestion
- Fix the issue of ledger rollover scheduled tasks were executed before reaching the ledger maximum rollover time [#11116](https://github.com/apache/pulsar/pull/11116)
```

File: site2/website/release-notes.md (hunk 2 above)
Comment (hangc0276) [reply site2/website/release-notes.md]: done

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**